handlers/officerDeltaHandler.go

```diff
@@ -28,34 +28,36 @@ func NewOfficerDeltaHandler(kSvc services.KafkaService, h helpers.Helper, chv va
 // encountered then they will be returned via the ResponseWriter.
 func (kp *OfficerDeltaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
-	log.Info(fmt.Sprintf("Open API spec to use: %s", kp.cfg.OpenApiSpec), nil)
+	contextId := r.Context().Value(helpers.XRequestId).(string)
+
+	log.InfoC(contextId, fmt.Sprintf("Open API spec to use: %s", kp.cfg.OpenApiSpec), nil)
```

Comment:
So instead of logging this way you can log the cfg.OpenApiSpec as a prt of data rather than passing  nil. And message can be Using the Open api spec.

Reply:
See for example - https://github.com/companieshouse/document-render-service/blob/master/registry/registry.go#L46

Reply:
Added the individual field as part of data rather than formatted string